diff --git a/build/config/phone/apps-production.list b/build/config/phone/apps-production.list
index 6e6e2e6..0cd2d7f 100644
--- a/build/config/phone/apps-production.list
+++ b/build/config/phone/apps-production.list
@@ -6,8 +6,9 @@ apps/callscreen
 apps/camera
 apps/clock
 apps/collection
-apps/communications
+apps/contacts
 apps/costcontrol
+apps/dialer
 apps/email
 apps/emergency-call
 apps/findmydevice
