diff --git a/build/import-config.js b/build/import-config.js
index 9ae59c2..193c110 100644
--- a/build/import-config.js
+++ b/build/import-config.js
@@ -60,9 +60,8 @@ function generateManifest (webapp, gaia) {
 }
 
 // Generates the configuration to be used by the importing processes
-function generateConfig (app, destination, gaia) {
-  var configFile = utils.getFile(gaia.stageDir.path, destination,
-                                'config.json');
+function generateConfig (app, gaia) {
+  var configFile = utils.getFile(gaia.stageDir.path, 'config.json');
 
   var defaultConfig = {
     'defaultContactsOrder': 'givenName',
